src/Integration.Vsix.UnitTests/SonarLintTagger/TextBufferIssueTrackerTests.cs

```diff
@@ -369,7 +369,7 @@ public void WhenNoIssuesAreFound_ListenersAreUpdated()
 
         private void CheckErrorListRefreshWasRequestedOnce()
         {
-            mockSonarErrorDataSource.Verify(x => x.RefreshErrorList(), Times.Once);
+            mockSonarErrorDataSource.Verify(x => x.RefreshErrorList(It.IsAny<SnapshotFactory>()), Times.Once);
```

Comment:
Does it have to be Any? We should be able to verify against the relevant factory.